nipype/interfaces/fsl/preprocess.py

```diff
@@ -113,7 +113,7 @@ class BETOutputSpec(TraitedSpec):
 
 
 class BET(FSLCommand):
-    """Use FSL BET command for skull stripping.
+    """FSL BET command for skull stripping
```

Comment:
we could use the nipype term `interface` or `wrapper`

Reply:
Just choose ;-)  I saw such phrasing used in other ones, and decided to go with it.  The point that it is an interface could somewhat be deduced from the module path, but I would also kinda like that being explicit, but then making it explicit that it is interface to the command (instead of to e.g. Python function/module) also has value.
In my case, I will use it for description of a flywheel gear, and there it would not matter either it is "interface" or "wrapper" (somewhat assumed), so I would need to strip it away or replace to something like above

Reply:
let's use `wrapper`